TTL: Error creating index An equivalent index already exists,

## Expected Behavior (Mandatory)
No error logs created when restarting an instance (either docker or other)

## Actual Behavior (Mandatory)
When restarting an instance which has `apoc.ttl.enabled=true` configured an error and traceback is generated when it tries to re-create the TTL index.  This can probably be resolved by ensuring the index is created using a CREATE OR REPLACE or a CREATE IF NOT EXISTS (where possible).

## How to Reproduce the Problem

1. startup a docker container image of the DB with `apoc.ttl.enabled=true`
1. wait for it to finish starting up
1. shut down the container
2. start up the same container again

### Simple Dataset (where it's possibile)

No cypher query needed

### Steps (Mandatory)

  1. See above

## Specifications (Mandatory)

Nil

Currently used versions

### Versions

  - OS:  Ubuntu 19.04
  - Docker: 19.03.11
  - Neo4j: 4.1.0
  - Neo4j-Apoc: 4.1.0.0

### Logs


```
neo4j_1                    | 2020-07-10 04:57:19.534+0000 ERROR [neo4j] TTL: Error creating index An equivalent index already exists, 'Index( id=1, name='index_a5e3e466', type='GENERAL BTREE', schema=(:TTL {ttl}), indexProvider='native-btree-1.0' )'.
neo4j_1                    | org.neo4j.graphdb.QueryExecutionException: An equivalent index already exists, 'Index( id=1, name='index_a5e3e466', type='GENERAL BTREE', schema=(:TTL {ttl}), indexProvider='native-btree-1.0' )'.
neo4j_1                    |    at org.neo4j.kernel.impl.query.QueryExecutionKernelException.asUserException(QueryExecutionKernelException.java:35)
neo4j_1                    |    at org.neo4j.cypher.internal.javacompat.ResultSubscriber.converted(ResultSubscriber.java:416)
neo4j_1                    |    at org.neo4j.cypher.internal.javacompat.ResultSubscriber.assertNoErrors(ResultSubscriber.java:397)
neo4j_1                    |    at org.neo4j.cypher.internal.javacompat.ResultSubscriber.init(ResultSubscriber.java:85)
neo4j_1                    |    at org.neo4j.cypher.internal.javacompat.ExecutionEngine.executeQuery(ExecutionEngine.java:112)
neo4j_1                    |    at org.neo4j.kernel.impl.coreapi.TransactionImpl.execute(TransactionImpl.java:236)
neo4j_1                    |    at org.neo4j.kernel.impl.coreapi.TransactionImpl.execute(TransactionImpl.java:225)
neo4j_1                    |    at org.neo4j.kernel.impl.factory.GraphDatabaseFacade.executeTransactionally(GraphDatabaseFacade.java:162)
neo4j_1                    |    at org.neo4j.kernel.impl.factory.GraphDatabaseFacade.executeTransactionally(GraphDatabaseFacade.java:152)
neo4j_1                    |    at org.neo4j.kernel.impl.factory.GraphDatabaseFacade.executeTransactionally(GraphDatabaseFacade.java:140)
neo4j_1                    |    at apoc.ttl.TTLLifeCycle.createTTLIndex(TTLLifeCycle.java:69)
neo4j_1                    |    at org.neo4j.kernel.impl.scheduler.ScheduledJobHandle.lambda$new$0(ScheduledJobHandle.java:89)
neo4j_1                    |    at org.neo4j.kernel.impl.scheduler.ThreadPool.lambda$asCallable$1(ThreadPool.java:94)
neo4j_1                    |    at org.neo4j.kernel.impl.scheduler.ThreadPool.lambda$submit$0(ThreadPool.java:74)
neo4j_1                    |    at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
neo4j_1                    |    at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
neo4j_1                    |    at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
neo4j_1                    |    at java.base/java.lang.Thread.run(Thread.java:834)
neo4j_1                    | Caused by: org.neo4j.kernel.impl.query.QueryExecutionKernelException: An equivalent index already exists, 'Index( id=1, name='index_a5e3e466', type='GENERAL BTREE', schema=(:TTL {ttl}), indexProvider='native-btree-1.0' )'.
neo4j_1                    |    ... 17 more
neo4j_1                    | Caused by: org.neo4j.exceptions.CypherExecutionException: An equivalent index already exists, 'Index( id=1, name='index_a5e3e466', type='GENERAL BTREE', schema=(:TTL {ttl}), indexProvider='native-btree-1.0' )'.
neo4j_1                    |    at org.neo4j.cypher.internal.planning.ExceptionTranslatingQueryContext.addIndexRule(ExceptionTranslatingQueryContext.scala:124)
neo4j_1                    |    at org.neo4j.cypher.internal.runtime.interpreted.UpdateCountingQueryContext.addIndexRule(UpdateCountingQueryContext.scala:113)
neo4j_1                    |    at org.neo4j.cypher.internal.SchemaCommandRuntime$$anonfun$1.$anonfun$applyOrElse$28(SchemaCommandRuntime.scala:140)
neo4j_1                    |    at org.neo4j.cypher.internal.SchemaCommandRuntime$$anonfun$1.$anonfun$applyOrElse$28$adapted(SchemaCommandRuntime.scala:137)
neo4j_1                    |    at org.neo4j.cypher.internal.procs.SchemaWriteExecutionPlan.run(SchemaWriteExecutionPlan.scala:54)
neo4j_1                    |    at org.neo4j.cypher.internal.CypherCurrentCompiler$CypherExecutableQuery.innerExecute(CypherCurrentCompiler.scala:333)
neo4j_1                    |    at org.neo4j.cypher.internal.CypherCurrentCompiler$CypherExecutableQuery.execute(CypherCurrentCompiler.scala:291)
neo4j_1                    |    at org.neo4j.cypher.internal.ExecutionEngine.doExecute(ExecutionEngine.scala:219)
neo4j_1                    |    at org.neo4j.cypher.internal.ExecutionEngine.$anonfun$executeSubQuery$1(ExecutionEngine.scala:180)
neo4j_1                    |    at org.neo4j.cypher.internal.ExecutionEngine.closing(ExecutionEngine.scala:185)
neo4j_1                    |    at org.neo4j.cypher.internal.ExecutionEngine.executeSubQuery(ExecutionEngine.scala:177)
neo4j_1                    |    at org.neo4j.cypher.internal.ExecutionEngine.execute(ExecutionEngine.scala:127)
neo4j_1                    |    at org.neo4j.cypher.internal.javacompat.ExecutionEngine.executeQuery(ExecutionEngine.java:122)
neo4j_1                    |    at org.neo4j.cypher.internal.javacompat.ExecutionEngine.executeQuery(ExecutionEngine.java:111)
neo4j_1                    |    ... 13 more
neo4j_1                    | Caused by: org.neo4j.kernel.api.exceptions.schema.EquivalentSchemaRuleAlreadyExistsException: An equivalent index already exists, 'Index( id=1, name='index_a5e3e466', type='GENERAL BTREE', schema=(:TTL {ttl}), indexProvider='native-btree-1.0' )'.
neo4j_1                    |    at org.neo4j.kernel.impl.newapi.Operations.assertNoBlockingSchemaRulesExists(Operations.java:1058)
neo4j_1                    |    at org.neo4j.kernel.impl.newapi.Operations.indexCreate(Operations.java:835)
neo4j_1                    |    at org.neo4j.kernel.impl.newapi.Operations.indexCreate(Operations.java:851)
neo4j_1                    |    at org.neo4j.kernel.impl.newapi.Operations.indexCreate(Operations.java:843)
neo4j_1                    |    at org.neo4j.kernel.impl.api.RestrictedSchemaWrite.indexCreate(RestrictedSchemaWrite.java:76)
neo4j_1                    |    at org.neo4j.cypher.internal.runtime.interpreted.TransactionBoundQueryContext.addIndexRule(TransactionBoundQueryContext.scala:743)
neo4j_1                    |    ... 27 more
```

--- a/full/src/main/java/apoc/ttl/TTLLifeCycle.java
+++ b/full/src/main/java/apoc/ttl/TTLLifeCycle.java
@@ -66,7 +66,7 @@ public class TTLLifeCycle extends LifecycleAdapter {
 
     public void createTTLIndex() {
         try {
-            db.executeTransactionally("CREATE INDEX ON :TTL(ttl)");
+            db.executeTransactionally("call apoc.schema.assert({ TTL: ['ttl'] }, null, false)");
         } catch (Exception e) {
             log.error("TTL: Error creating index", e);
         }

